pallets/vector/src/lib.rs

```diff
@@ -451,11 +463,11 @@ pub mod pallet {
 		/// Executes message if a valid proofs are provided for the supported message type, assets and domains.
 		#[pallet::call_index(1)]
 		#[pallet::weight({
-			match addr_message.message {
-				Message::ArbitraryMessage(ref data) => T::WeightInfo::execute_arbitrary_message(data.len() as u32),
-				Message::FungibleToken {..} => T::WeightInfo::execute_fungible_token(),
-			}
-		})]
+        match addr_message.message {
+        Message::ArbitraryMessage(ref data) => T::WeightInfo::execute_arbitrary_message(data.len() as u32),
+        Message::FungibleToken {..} => T::WeightInfo::execute_fungible_token(),
+        }
+        })]
```

Comment:
This is not looking good

Reply:
Can't help it

Reply:
but it was ok before, not important tho

Reply:
This was not changed in this pr, was it changed before?

Reply:
Oh, you're talking about formatting, I misunderstood it. cargo fmt is not helpful so I will revert it.